package/src/StreamMediaCache.ts

```diff
@@ -0,0 +1,72 @@
+import RNFS from 'react-native-fs';
```

Comment:
i didnt create a singleton for this one cause i had no need for it.

do you think its a good idea to expose the API to users? @vishalnarkhede

Reply:
Nah I think this is fine. Lets expose later if anyone asks for it!!